README.md

```diff
@@ -42,18 +42,20 @@ This is quite slow and can take up to one or two hours but can provide good resu
 -   Go
 -   PHP
 -   TypeScript
+-   TSX
 -   Java
 -   C#
 -   C++
 -   Python
--   JavaScript
+-   JavaScript (without @flow annotation)
```

Comment:
I would keep it as "JavaScript" and add "Flow-annotated JavaScript" to the "Basic Support". Otherwise, we have to mention also other JavaScript dialects here that we might not support.